runtime/parachains/src/disputes.rs

```diff
@@ -476,6 +474,9 @@ pub mod pallet {
 	/// fewer than `byzantine_threshold + 1` validators.
 	///
 	/// The i'th entry of the vector corresponds to the i'th validator in the session.
+	#[deprecated(
```

Comment:
I'm not sure if this is the correct way to remove a storage item.

The migration will work with the new runtime so the item should exist somewhere. The [substrate documentation](https://docs.substrate.io/build/upgrade-the-runtime/) gives [this example](https://github.com/hicommonwealth/substrate/blob/5f3933f5735a75d2d438341ec6842f269b886aaa/frame/indices/src/migration.rs#L5-L22). It looks like the storage item is declared only in the migration code but I'm not sure if this means that it can be removed from the pallet.

Help is much appreciated here.

Reply:
Actually this is pointless. The scope where this code is located is irrelevant (because it works with the underlying database) so it can be in the migration module in a `mod depricated{}` - as in the substrate example.

Reply:
There's not much need to mark this as deprecated. No one other than this team uses this runtime pallet, right? 

You can :
* remove the storage item, 
* clear it manually in the migration 
* forget it about ;) 


here's an example that roughly does something similar (removal of `HistoryDepth`) https://github.com/paritytech/substrate/pull/12230/files#diff-cbe1855fd3bef9c6878a793c91771dd0013079a705b0ab1445273ea70dc3b25fL246, I am sure the author @Ank4n would also have good experience to share and can guide you here.

Reply:
Thanks a ton!

Reply:
Done as suggested! Thank you @kianenigma !